Description: Remove build options not necessary for Debian
 Do not duplicate all the Debian build flags.
 Stripping is done by dh_strip separately.
Author: James Cowgill <jcowgill@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -347,20 +347,6 @@ else ()
     set ( DEBUG_BUILD 0 )
     set(CMAKE_INSTALL_DO_STRIP TRUE)
     message("-- CMAKE_INSTALL_DO_STRIP is " ${CMAKE_INSTALL_DO_STRIP})
-    if(UNIX AND NOT APPLE)
-        ## Avoid hardening-no-relro wrarning messages from lintian
-        if(EXISTS "/usr/bin/dpkg-buildflags")
-            execute_process(COMMAND /usr/bin/dpkg-buildflags --get CFLAGS OUTPUT_VARIABLE EXTRA_CFLAGS OUTPUT_STRIP_TRAILING_WHITESPACE ERROR_QUIET)
-            execute_process(COMMAND /usr/bin/dpkg-buildflags --get CPPFLAGS OUTPUT_VARIABLE EXTRA_CPPFLAGS OUTPUT_STRIP_TRAILING_WHITESPACE ERROR_QUIET)
-            execute_process(COMMAND /usr/bin/dpkg-buildflags --get CXXFLAGS OUTPUT_VARIABLE EXTRA_CXXFLAGS OUTPUT_STRIP_TRAILING_WHITESPACE ERROR_QUIET)
-            execute_process(COMMAND /usr/bin/dpkg-buildflags --get LDFLAGS OUTPUT_VARIABLE EXTRA_LDFLAGS OUTPUT_STRIP_TRAILING_WHITESPACE ERROR_QUIET)
-            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${EXTRA_CXXFLAGS}")
-            add_definitions(${EXTRA_CPPFLAGS})
-            set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${EXTRA_CFLAGS}")
-            set(LINKER_OPTIONS "${EXTRA_LDFLAGS}")
-            list(APPEND LINKER_OPTIONS "-s") ## strip binaries
-        endif()
-    endif()
 
     set( CL_PCH_FILE "${CL_SRC_ROOT}/PCH/precompiled_header_release.h")
     set( CL_PCH_TARGET "precompiled_header_release.h.gch")
@@ -381,9 +367,6 @@ else ()
     message("-- Shared Objects will be written into ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
 
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")        ## Optimize
-    if(CMAKE_COMPILER_IS_GNUCXX)
-        set(CMAKE_EXE_LINKER_FLAGS "-s")  ## Strip binary
-    endif(CMAKE_COMPILER_IS_GNUCXX)
 
     ## In release, add NDEBUG unless explicitly told not to
     if (NOT PREVENT_WX_ASSERTS MATCHES 0)
